### PR TITLE
Fix build failure

### PR DESCRIPTION
**Pull request by vhirtham**
## Changes

Fix the current [build failure](https://github.com/BAMWelDX/weldx/runs/7280019473?check_suite_focus=true)

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -37,7 +37,7 @@ setup_requires =
     setuptools >=38.3.0
     setuptools_scm
 install_requires =
-    numpy >=1.20
+    numpy ==1.21
```
Comment (vhirtham) [inline setup.cfg:40]:
```suggestion
numpy >=1.20
```

File: setup.cfg [hunk 2]
```diff
@@ -37,7 +37,7 @@ setup_requires =
     setuptools >=38.3.0
     setuptools_scm
 install_requires =
-    numpy >=1.20
+numpy >=1.20
```
Comment (vhirtham) [inline setup.cfg:40]:
```suggestion
    numpy >=1.20
```

File: .github/workflows/build_pkg.yml [hunk 3]
```diff
@@ -53,7 +53,7 @@ jobs:
         run: conda list
 
       - name: Build from local conda recipe
-        run: conda mambabuild .
+        run: conda mambabuild -c conda-forge .
```
Comment (CagtayFabry) [inline .github/workflows/build_pkg.yml:56]: I am not sure why the previously available channels are no longer detected here but this seems to fix the build issues

**Review by CagtayFabry — APPROVED** [review]

File: .github/workflows/build_pkg.yml (hunk 3 above)
Comment (vhirtham) [reply .github/workflows/build_pkg.yml:56]: Interesting. I thought it was covered by the recipe that contains conda-forge. Good that you found a solution. Didn't know what to try else.

**Pull request merged**